cmake/ClangTools.cmake

```diff
@@ -0,0 +1,10 @@
+find_program(CLANG_FORMAT NAMES "clang-format")
+
+if(CLANG_FORMAT)
+  message(STATUS "clang-format found: -i -style=file ${tfs_HDR}")
+  add_custom_target(
+    clang-format
+    COMMAND ${CLANG_FORMAT}
+    -i -style=file ${tfs_SRC} ${tfs_HDR}
+    )
+endif()
```

Comment:
I don't think you should check for clang-format executable. Just let it fail if the executable doesn't exist.